packages/lodestar/src/api/impl/validator/index.ts

```diff
@@ -161,7 +161,19 @@ export function getValidatorApi({
       await waitForSlot(slot); // Must never request for a future slot > currentSlot
 
       const headRoot = chain.forkChoice.getHeadRoot();
-      const state = await chain.regen.getBlockSlotState(headRoot, slot);
+
+      const mChildLabels = {entrypointFn: "getBlockSlotState", callingModule: "produceAttestationData"};
+
+      const state = await chain.regen.getBlockSlotState(
+        headRoot,
+        slot,
+        metrics
+          ? {
+              stateLookupsTotal: metrics.regenStateCacheLookupTotal.child(mChildLabels),
```

Comment:
You are instantiating a child metrics instance on each call. Instantiation should only occur once. The .child pattern is probably not the right solution here